libraries/core_libs/consensus/src/vote_manager/vote_manager.cpp

```diff
@@ -471,6 +490,179 @@ uint64_t VoteManager::roundDeterminedFromVotes(size_t two_t_plus_one) {
   return 0;
 }
 
+void VoteManager::sendRewardPeriodCertVotes(uint64_t reward_period) {
```

Comment:
also this function should not be part of vote_mngr, I think it should be part of pbft as it is only used there

Reply:
Even the function only used on PBFT manager, I prefer all votes operation in vote manager to keep PBFT manager as much as simple